src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/geoMap.js

```diff
@@ -65,6 +65,25 @@ function makeGenerator() {
 
 const makeId = makeGenerator()
 
+class MapController extends deck.MapController {
+    handleEvent(event) {
+        if (event.type === 'wheel') {
+            if (!event.srcEvent.ctrlKey) {
+                // this.controllerStateProps.latitude += event.srcEvent.deltaX
+                // this.controllerStateProps.longitude += event.srcEvent.deltaY
+                // this.updateViewport(this.controllerStateProps, {transitionDuration: 0}, {
+                //     isDragging: true,
+                //     isPanning: true
+                // });
```

Comment:
commented code

Reply:
Also, this whole code block should probably be `if (combined_cond) { super.handleEvent(event) }` if the commented code is no longer relevant.

Reply:
If `GeoMap` is not fixed here, there probably should be no changes at all in this file.

Reply:
@MichaelMauderer I left it as super.handleEvent so that it will be easier for me or anyone to start working on next issue, knowing where to go.